src/components/vedtak/VedtakOppsummering.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import {
+    Column,
+    Row,
+} from 'nav-frontend-grid';
+import {
+    Normaltekst,
+    Undertittel,
+} from 'nav-frontend-typografi';
+import { restdatoTildato } from '../../utils/datoUtils';
+import { VedtakDTO } from '../../reducers/vedtak';
+import styled from 'styled-components';
+
+const texts = {
+    oppsummering: 'Oppsummering',
+    maksdato: 'Maksdato',
+    vedtaksdato: 'Vedtaksdato',
+    dagerGjenstar: 'Dager gjenstår',
+    dagerBrukt: 'Dager brukt hittil',
+    totalSykepengedager: 'Sykepengedager totalt',
```

Comment:
Det er sikkert i henhold til skissene, men hvorfor viser vi totalt antall sykepengedager? Det vil jo alltid være det samme for alle?